test/gst-va/encode/vp9.py

```diff
@@ -0,0 +1,99 @@
+###
+### Copyright (C) 2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.va.util import *
+from ....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "8bit")
+
+@slash.requires(*have_gst_element("vavp9dec"))
+class VP9EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "vp9",
+      gstdecoder    = "vavp9dec",
+      gstmediatype  = "video/x-vp9",
+      gstparser     = "vp9parse",
+      gstmuxer      = "matroskamux",
+      gstdemuxer    = "matroskademux",
+    )
+
+  def get_file_ext(self):
+    return "webm"
+
+@slash.requires(*platform.have_caps("vdenc", "vp9_8"))
+@slash.requires(*have_gst_element("vavp9lpenc"))
+class VP9EncoderLPTest(VP9EncoderBaseTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      caps          = platform.get_caps("vdenc", "vp9_8"),
+      lowpower      = True,
+      gstencoder    = "vavp9lpenc",
+    )
+
+class cqp_lp(VP9EncoderLPTest):
+  def init(self, tspec, case, ipmode, qp, quality, looplvl, loopshp):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = 30 if ipmode != 0 else 1,
+      looplvl   = looplvl,
+      loopshp   = loopshp,
+      quality   = quality,
+      rcmode    = "cqp",
+      qp        = qp,
+    )
+
+  @parametrize_with_unused(*gen_vp9_cqp_lp_parameters(spec), ['refmode'])
```

Comment:
`slices` should not have been removed here (only for the non-lp tests).  It is in the test parameters for the lp tests but it is not used.  Thus, it needs to be announced here.

Reply:
right, my mistake, added slices back for lp mode